app/src/main/java/com/mozilla/hackathon/kiboko/Analytics.java

```diff
@@ -16,26 +14,22 @@
 import java.util.Date;
 import java.util.List;
 
-/**
- * Created by secretrobotron on 7/2/16.
- */
-
 public class Analytics {
 
     private static final String ANALYTICS_FILENAME = ".jisort_analytics.txt";
     private static final String ANALYTICS_ARCHIVE_FILENAME = ".jisort_analytics.1.txt";
-    private static final long TIME_BETWEEN_SAVES= 5000;
+    private static final long TIME_BETWEEN_SAVES = 5000;
     private static final long FILE_SIZE_LIMIT = 100000; //bytes
 
     public static void shareAnalytics() {
         Analytics.get().share();
     }
 
-    private void share () {
+    private void share() {
         File currentFile = new File(Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_DOWNLOADS), ANALYTICS_FILENAME);
         File oldFile = new File(Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_DOWNLOADS), ANALYTICS_ARCHIVE_FILENAME);
 
-        ArrayList<Uri> files = new ArrayList<Uri>();
```

Comment:
Android has had support for diamond operators since March 2014, it's a Java 7 language feature.